types/Cfgu.ts

```diff
@@ -1,40 +1,40 @@
 export type CfguType =
```

Comment:
I'm guessing your IDE did this? Let's not change this 🙏🏻

Reply:
Sure. sorry